presto-spi/src/main/java/io/prestosql/spi/security/AccessDeniedException.java

```diff
@@ -321,6 +326,11 @@ public static void denyCreateViewWithSelect(String sourceName, ConnectorIdentity
         throw new AccessDeniedException(format("View owner '%s' cannot create view that selects from %s%s", identity.getUser(), sourceName, formatExtraInfo(extraInfo)));
     }
 
+    public static void denySelectFromColumns(AccessDeniedException e)
+    {
+        throw new AccessDeniedException("View owner does not have sufficient privileges", e);
```

Comment:
this seems generic and not specific to columns only. All column names available (accessible) to query user?

Reply:
Thank you, renamed.

Reply:
I'm not sure about column names. Visibility of column names are out of scope of this pr.